maven/apichanges.xml

```diff
@@ -83,6 +83,21 @@ is the proper place.
     <!-- ACTUAL CHANGES BEGIN HERE: -->
 
     <changes>
+        <change id="jpda-attach-trigger">
+           <api name="general"/>
+           <summary>Attach to Debugger on triggers</summary>
+           <version major="2" minor="114"/>
+           <date day="16" month="11" year="2017"/>
+           <author login="monacotoni"/>
+           <compatibility modification="yes" semantic="compatible"/>
+           <description>
+               <p>
+                   New property to allow  to attach debugger at a certain time: 
+                   <code>ACTION_PROPERTY_JPDAATTACH_TRIGGER</code>
```

Comment:
It is good idea to add a link to the anchor generated from the `arch.xml` change. E.g. link from here to the actual description of the API. The href would look like `@TOP@architecture-overview.html#.....`, I think.